src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dAdapter.java

```diff
@@ -4,14 +4,12 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+//import edu.kis.legacy.drawer.panel.DefaultDrawerFrame;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
 
-/**
- * driver adapter to drawer with several bugs.
- */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class Job2dAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it does not indicate the purpose of the adapter and it lacks a driver aspect.
Inheritance is not used (no super drawLine calls). Should be removed!